src/main/resources/hudson/plugins/s3/S3ArtifactsProjectAction/jobMain.jelly

```diff
@@ -1,3 +1,4 @@
+<?jelly escape-by-default='true'?>
```

Comment:
`${h.escape(artifact.name)}` below is no longer required, I'd guess

Reply:
Probably not. At worst this would double-escape I think.